### ISSUE TITLE
The spectrum files are not present in zip file

### ISSUE THREAD

**Issue by umutcakir** [compomics/peptide-shaker#432]
Hello, 

I run peptide shaker with following command: 
java -cp PeptideShaker-2.0.2/PeptideShaker-2.0.2.jar eu.isas.peptideshaker.cmd.PeptideShakerCLI -reference Reference -identification_files "X/InputforPeptideshaker/" -zip "X/OutputDirectory/peptideshakerfile.zip" 

Then in order to generate reports I used following command:
java -cp PeptideShaker-2.0.2/PeptideShaker-2.0.2.jar eu.isas.peptideshaker.cmd.ReportCLI -in "X/OutputDirectory/peptideshakerfile.zip" -out_reports "X/OutputforReports/" -reports "0"

But ReportCLI gives following error:
"The spectrum files were not found. Please provide their location in the command line parameters." 

I also checked peptideshakerfile.zip whether it contains the spectrum files, but only .csm files .fmi and .fasta .psdb files are present. I also used .psdb file for ReportCLI (after unzip peptideshakerfile.zip manually) but again it gives the same error message. How can I include the spectrum files in output of PeptideShakerCLI automatically. If it is not possible now, how can I provide spectrum files location for ReportCLI. 

Note that X is full path, but shorten here just for simplicity. 

Thank you for your help.

**Comment by hbarsnes** [comment]
As far as I can tell the problem is that the PeptideShaker zip file does not contain the psdb file itself given that you are not using the -out option for the PeptideShakerCLI command line. Can you try adding -zip "X/OutputDirectory/peptideshakerfile.psdb" to your PeptideShakerCLI command line?

Note that you can also attach the ReportCLI options directly on the PeptideShakerCLI command line, skipping the -in "X/OutputDirectory/peptideshakerfile.zip" option of course. That way you get away with one command line and do not have to reopen the psdb file just to do the export.

**Comment by umutcakir** [comment]
Thank you for your prompt reply.

Adding -zip "X/OutputDirectory/peptideshakerfile.psdb" line does not work. When I try to open generated psdb file for report, following message was logged:
"org.sqlite.SQLiteException: [SQLITE_NOTADB]  File opened that is not a database file (file is encrypted or is not a database)" 

On the other hand, I tried to open example dataset (HeLa Example.psdb) for report, following message was logged (I tried in two server separately): 
"java.io.IOException: FASTA file not found C:\Users\Harald\Desktop\Updated tutorials\november-2020\data\uniprot-human-reviewed-trypsin-november-2020_concatenated_target_decoy.fasta." 

Command line for this is following:
java -cp PeptideShaker-2.0.2/PeptideShaker-2.0.2.jar eu.isas.peptideshaker.cmd.ReportCLI -in "X/PeptideShaker-2.0.2/resources/example_dataset/HeLa Example.psdb" -out_reports "X/OutputforReports/" -reports "0"

I think all problems are related with addresses of fasta/spectrum files. Does my command line for opening example dataset have a missing argument(s) and is there anything else that I can do to solve that problem?

**Comment by hbarsnes** [comment]
> Adding -zip "X/OutputDirectory/peptideshakerfile.psdb" line does not work

I assume you mean -out "X/OutputDirectory/peptideshakerfile.psdb"? And have you confirmed that the psdb file is a valid PeptideShaker project?

> On the other hand, I tried to open example dataset (HeLa Example.psdb) for report, following message was logged [...]

This one I can reproduce on my end. I will look into it and get back to you.

**Comment by hbarsnes** [comment]
I've just released PeptideShaker v2.0.4 which should solve the problem of not locating the FASTA and spectrum files.

It will look for these in following three locations: 1) the original location(s) used when creating the project, 2) in the same folder as the psdb file, and 3) in a subfolder called "data" in the folder containing the psdb file. (If a valid PeptideShaker zip file is used as input, the data subfolder will already be included and those files will be used.)

If you make sure that the required FASTA and spectrum files are in any of these locations the command lines should be able to locate them.

Note that I've also made the use of the out option mandatory when using the zip option on the PeptideShakerCLI command line.
